crates/ark/src/data_explorer/format.rs

```diff
@@ -49,7 +49,33 @@ pub fn format_string(x: SEXP, format_options: &FormatOptions) -> Vec<String> {
 }
 
 fn format(x: SEXP, format_options: &FormatOptions) -> Vec<FormattedValue> {
-    format_values(x, format_options).unwrap_or(unknown_format(x))
+    let mut formatted = format_values(x, format_options).unwrap_or(unknown_format(x));
+
+    // Truncate the values if they are too long
+    formatted.iter_mut().for_each(|v| {
+        if let FormattedValue::Value(x) = v {
+            truncate_inplace(x, format_options.max_value_length);
+        }
+    });
```

Comment:
I think I would vote for calling `truncate_inplace()` _inside_ of `format_values()`. That seems like it would make more sense to me?

Reply:
The reason for adding it here, is mostly to avoid adding truncation in multiple places as we have early returns in `format_values`. Since `format_values` is not public interface, seems fine to leave it here?